examples/weather/unified_recipe/train.py

```diff
@@ -153,22 +155,33 @@ def normalize_variables(variables, batch_norm):
     )
 
     # Initialize filesytem (TODO: Add multiple filesystem support)
-    fs = fsspec.filesystem("file")
+    fs = get_filesystem(
+        cfg.filesystem.type,
+        cfg.filesystem.key,
+        cfg.filesystem.endpoint_url,
+        cfg.filesystem.region_name,
+    )
 
     # Get filesystem mapper for datasets
     train_dataset_mapper = fs.get_mapper(cfg.curated_dataset.train_dataset_filename)
     val_dataset_mapper = fs.get_mapper(cfg.curated_dataset.val_dataset_filename)
 
     # Initialize validation datapipe
     val_datapipe = SeqZarrDatapipe(
-        zarr_dataset=zarr.open(val_dataset_mapper, mode="r"),
+        file_mapping=val_dataset_mapper,
```

Comment:
As I imagined, this is a `fsspec` mapper, is it appropriate to type it as a `str` without further documentation?

Reply:
Addressed above.